eo-maven-plugin/src/test/java/org/eolang/maven/VerifyMojoTest.java

```diff
@@ -100,19 +105,58 @@ void doesNotDetectWarningWithoutCorrespondingFlag(@TempDir final Path temp) {
     }
 
     @Test
-    @Disabled
-    void detectsCriticalError(@TempDir final Path temp) {
+    void failsOptimizationOnError(@TempDir final Path temp) {
```

Comment:
@levBagryansky Could you add human readable message for this assertion, please?
https://www.yegor256.com/2023/01/19/layout-of-tests.html#assertions